CouchPotato - Generic Service approach

Safihre opened the pull request:
### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] Package upgrade completed successfully
- [x] New installation of package completed successfully

ymartin59 commented on spk/couchpotatoserver/src/service-setup.sh:
```diff
@@ -0,0 +1,22 @@
+service_postinst ()
+{
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${SYNOPKG_PKGDEST}/env >> ${INST_LOG} 2>&1
```
OK for Python environment, but am I wrong or `pip install` from `requirements.txt` file is required and lacking ?

Safihre replied:
Couchpotato has no requirements, everything comes included :)

ymartin59 replied:
OK so in that specific case, is a pip environment really required ??

Safihre replied:
Not really, but it's considered best practice to make sure one package can't mess with anything. And doesn't take a lot of space so seems like a good idea always.

ymartin59 commented:
I will propose a french translation there too.

ymartin59 commented:
Please change Git clone URL to https://github.com/CouchPotato/CouchPotatoServer.git

ymartin59 commented on spk/couchpotatoserver/Makefile:
```diff
@@ -1,31 +1,32 @@
 DESCRIPTION_FRE = CouchPotato permet le téléchargement automatique de films via NZB ou torrent. Vous pouvez créer une \\\"liste de films à voir\\\", CouchPotato ira alors automatiquement rechercher les NZBs ou torrents correspondants toutes les X heures. Une fois qu\\\'une release correcte est trouvée, correspondant à la qualité demandé, elle sera ajouté a SABnzbd/NZBGet, ou téléchargée sous form ...
-DESCRIPTION_SPN = CouchPotato es un sistema automático de descarga de peliculas via NZB y torrent. Puedes tener una \"lista de películas para ver\" y el sistema buscará automáticamente cada X horas. Cuando el release es encotrado con la calidad que especificaste el sistema la enviará al programa de descargas SABnzbd/NZBGet/Transmision o descargá el .nzb o .torrent al directorio que especifiques.  ...
-ADMIN_PORT = 5050
+DESCRIPTION_SPN = CouchPotato es un sistema automático de descarga de peliculas via NZB y torrent. Puedes tener una \"lista de películas para ver\" y el sistema buscará automáticamente cada X horas. Cuando el release es encotrado con la calidad que especificaste el sistema la enviará al programa de descargas SABnzbd/NZBGet/Transmision o descargá el .nzb o .torrent al directorio que especifiques.  ...
```
I just wonder why spain translation `DESCRIPTION_SPN` is marked as changed

Safihre replied:
I think something with whitespace probably.

ymartin59 replied:
Right. Trailing white space was removed

Safihre commented:
@ymartin59 Will wait for your French translations then and update all at once. Maybe you can post them in the Sabnzbd PR since most texts overlap and I can do 1 bit update of the PRs.

Safihre commented on spk/couchpotatoserver/Makefile:
```diff
@@ -1,31 +1,32 @@
-DSM_UI_DIR = app
 
 DEPENDS =
 SPK_DEPENDS = "python>2.7.3-3:git"
```
I would highly doubt it, but maybe we can ask the people reporting problems now to test the new packages? (since we already have their attention)
This way we get feedback and know for sure.

Safihre commented:
@ymartin59 I incorporated the `sc-media` -> `sc-download` code into this PR.
I also found that in the `service.installer` some `if` statements had at the end `&> /dev/null`, so they would always return empty, although it didn't really affect the results it seems. 
I fixed that too.

This PR doesn't really depend on anything, so if this is merged I can incorporate the same code into Sonarr/Radarr/Sickbeard/Sickbeard-custom and we can get things published again!

Safihre commented:
@ymartin59 did you have time to look at this code that I added for dealing with the `sc-media`?

Safihre commented:
Tested this extensively today and turns out the code was pretty wrong.
Corrected it now and applied it to Couchpotato, Radarr and Sonarr.
After this is merged, I will also add it to #3093 for Sickrage and Sickbeard-custom.

Safihre commented:
I would like this to be reviewed and merged so I can continue converting packages..
